compile-front-end-packages.sh

```diff
@@ -66,7 +66,6 @@ for MODULE_DIR in `ls`
 do
   pushd .
   cd $MODULE_DIR
-  update_npm_package
```

Comment:
Are you sure we don't need this `update` here?

Reply:
Yea, actually, we are running this update_npm_package to update the front end component to the latest version without modifying manually, but this commond didn't consider version matching issues. In `module` directory, we only need to update the related "@ebi-gene-expression-group/atlas-homepage-cards" pakcage, but not "@ebi-gene-expression-group/eslint-config" and etc. In this way, we will not cause version dismatch issues under `MODULE_DIR`. And of course, if we would like to update "@ebi-gene-expression-group/atlas-homepage-cards" pakcage, we need to do that manually and keep an eye on the version compability issues.

Reply:
I have another comment here, I could not remember why we had this module directory ( but I think it had some reasons), but we may also be able to move all module packages as published npm package as other front end components, so we don't have to build any local front end component under `atlas-web-single-cell`, in this way, we would not have any package version issues while building bundles, as we should sort it out in the front end before integrating it to the back end. I suggest to do some refactoring not only for the [decoupling experiment page](https://github.com/ebi-gene-expression-group/atlas-components/issues/171) issue, but also the module directory.

Reply:
OK. If it is simplifying how to build the app then I am all in favour.